### PR TITLE
Improve error handling on InvalidRequestError

### PR DESCRIPTION
**Pull request by kss-espeo**

### CODE REVIEW

File: src/domain/request/usecase/ExecuteReadyRequestsUseCase.ts [hunk 1]
```diff
@@ -56,11 +55,13 @@ class ExecuteReadyRequestsUseCase {
         request.getSelectionPath(),
       );
       response.addSelectedData(selectedData);
+      this.logger.info(`Sending response [response=${JSON.stringify(response)}]`);
```
Comment (starspire) [inline src/domain/request/usecase/ExecuteReadyRequestsUseCase.ts]: This method (`fetchAndSelectData`) doesn't know anything about sending responses. You should rename logs content.

Comment (kss-espeo) [reply src/domain/request/usecase/ExecuteReadyRequestsUseCase.ts]: Any specific idea? I just moved this line, didn't write so not sure what was intention of an author.

Comment (tcichowicz) [reply src/domain/request/usecase/ExecuteReadyRequestsUseCase.ts]: You're in a `fetchAndSelectData` function so how about 
```suggestion
      this.logger.info(`Fetched and selected data [response=${JSON.stringify(response)}]`);
```

File: src/domain/request/usecase/ExecuteReadyRequestsUseCase.ts [hunk 2]
```diff
@@ -56,11 +55,13 @@ class ExecuteReadyRequestsUseCase {
...
 
       return response;
     } catch (e) {
       if (e instanceof InvalidRequestError) {
         response.setError(e.code);
+        this.logger.error(`Sending response [response=${JSON.stringify(response)}]`, e);
```
Comment (tcichowicz) [inline src/domain/request/usecase/ExecuteReadyRequestsUseCase.ts]: ditto ;)

File: src/domain/request/usecase/ExecuteReadyRequestsUseCase.ts [hunk 3]
```diff
@@ -56,11 +55,13 @@ class ExecuteReadyRequestsUseCase {
         request.getSelectionPath(),
       );
       response.addSelectedData(selectedData);
+      this.logger.info(`Fetched and selected data [response=${JSON.stringify(response)}]`);
 
       return response;
     } catch (e) {
       if (e instanceof InvalidRequestError) {
         response.setError(e.code);
+        this.logger.error(`Fetched and selected data [response=${JSON.stringify(response)}]`, e);
```
Comment (tcichowicz) [inline src/domain/request/usecase/ExecuteReadyRequestsUseCase.ts]:
```suggestion
        this.logger.error(`Could not fetch and/or select data [response=${JSON.stringify(response)}]`, e);
```

**Pull request merged**